src/components/html/EcApplication.html

```diff
@@ -2,19 +2,34 @@
   <a-row :gutter="24">
     <div v-if="isLoaded">
       <a-col :md="24" :lg="7">
-        <a-card :bordered="false" >
+        <a-card :bordered="true" >
           <div class="text-center">
-            <a-avatar :src="icon" :size="104" class="mb-3"/>
-            <div style="margin-bottom: 5px">{{ title }}</div>
+            <a-avatar :src="icon" :size="170"  class="mb-3 img-fluid"/>
```

Comment:
Ficou melhor forçando este size 170?

Reply:
Sim, antes o logotipo estava muito pequeno em 104